config/adminlte.php

```diff
@@ -79,6 +79,18 @@
 
     /*
     |--------------------------------------------------------------------------
+    | Enable Gate Checking
+    |--------------------------------------------------------------------------
+    |
+    | Simple authorization checking to remove menu items based on user
+    | permission. Set menu item 'permission' to the permission that
+    | you would like to check for.  Must be  implemented by Gate
+    |
+    */
+   'check_auth'=>false,
```

Comment:
I don't think a configuration item is needed here. It would be better to remove this feature and just depend on whether it is configured via the menu items.